website/docs/guides/creating-pages.md

```diff
@@ -18,6 +19,12 @@ Pages do not have sidebars, only [docs](./docs/docs-introduction.md) do.
 
 :::
 
+:::info
```

Comment:
Maybe move this down to the section `## Add a Markdown page`?

Reply:
For consistency I added this link at the same place in all docs/blog/content guides. 

Also this does not make sense to add this to "markdown page" because it links to plugin API ref, not the just frontmatter options so this applies to the react pages as well.